containers/covabot/src/cova-bot/__tests__/enhanced-llm-triggers.test.ts

```
@@ -0,0 +1,431 @@
+
+    try {
+      const response = await this.llmService.createSimpleCompletion(userPrompt, systemPrompt);
+      return response.trim();
+    } catch (error) {
+      logger.error('[EnhancedLLM] Real LLM service failed:', error as Error);
+      throw error; // Will trigger fallback in generateResponse
+    }
+  }
+
+  isUsingRealLLM(): boolean {
+    return this.llmService && this.llmService.isInitialized();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)